### ISSUE TITLE
Restore error on winui-net6 template

### ISSUE THREAD

**Issue by PhoenixWyllow** [unoplatform/uno#8775]
### Current behavior

After updating the uno.check tool to 1.4.0-dev4 and running the tool with the dev flag I created a new app from the terminal.
I ran this command: `dotnet new unoapp-winui-net6 -o MacroDeck3`
This was the result:
```
The template "Multi-Platform App net6 (WinUI)" was created successfully.

Processing post-creation actions...
Running 'dotnet restore' on E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.Shared\MacroDeck3.Shared.shproj...
Restore succeeded.
Running 'dotnet restore' on E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.Mobile\MacroDeck3.Mobile.csproj...
  Determining projects to restore...
  Restored E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.Mobile\MacroDeck3.Mobile.csproj (in 11.1 sec).
Restore succeeded.
Running 'dotnet restore' on E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.UWP\MacroDeck3.Uwp.csproj...
Restore succeeded.
Running 'dotnet restore' on E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.Windows\MacroDeck3.Windows.csproj...
  Determining projects to restore...
  Restored E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.Windows\MacroDeck3.Windows.csproj (in 20.93 sec).
Restore succeeded.
Running 'dotnet restore' on E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.macOS\MacroDeck3.Skia.Gtk.csproj...
MSBUILD : error MSB1009: Project file does not exist.
Switch: E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.macOS\MacroDeck3.Skia.Gtk.csproj
Restore failed.
Running 'dotnet restore' on E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.macOS\MacroDeck3.Skia.WPF.csproj...
MSBUILD : error MSB1009: Project file does not exist.
Switch: E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.macOS\MacroDeck3.Skia.WPF.csproj
Restore failed.
Running 'dotnet restore' on E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.macOS\MacroDeck3.Skia.WPF.Host.csproj...
MSBUILD : error MSB1009: Project file does not exist.
Switch: E:\Code\source\repos\MacroDeck\MacroDeck3\MacroDeck3.macOS\MacroDeck3.Skia.WPF.Host.csproj
Restore failed.
Post action failed.
Manual instructions: Run 'dotnet restore'
```

It seems that the paths are incorrect for Skia.Gtk, Skia.WPF and Skia.WPF.Host.

### Expected behavior

Restore should run successfully for all projects if the `dotnet restore` command is given the correct file paths.

### How to reproduce it (as minimally and precisely as possible)

- from terminal/powershell/cmd:
- run the command: `dotnet new unoapp-winui-net6 -o TestWinUINet6App`
- `MSBUILD : error MSB1009: Project file does not exist.` messages should appear for the Skia.* csproj files.
- `Restore failed.` should appear for the same project files.

### Workaround

Manually running `dotnet restore` with the correct paths for the Skia.* csproj files resolves this.

### Affected platforms

Solution Templates

### IDE

Visual Studio 2022

### IDE version

17.3 preview1

**Comment by jeromelaban** [comment]
Thanks for the report, it's indeed a template authoring error. It will be fixed by #8783.

**Comment by PhoenixWyllow** [comment]
wonderful to hear/read! Sorry I didn't PR this myself but I have very little knowledge regarding templates, etc. Glad to have been helpful in some way.
